connector/src/it/scala/com/datastax/spark/connector/cql/ContinuousPagingScannerSpec.scala

```diff
@@ -76,6 +83,40 @@ class ContinuousPagingScannerSpec extends SparkCassandraITFlatSpecBase with Defa
 
     withClue(sessions.map(_.toString).mkString("\n"))(sessions.size should be(1))
   }
+
+  private def executeContinuousPagingScan(readConf: ReadConf): Statement[_] = {
+    // we don't want to use the session from CC as mockito is unable to spy on a Proxy
+    val cqlSession = conn.conf.connectionFactory.createSession(conn.conf)
+    try {
+      val sessionSpy = spy(cqlSession)
+      val scanner = ContinuousPagingScanner(readConf, conn.conf, IndexedSeq.empty, sessionSpy)
+      val stmt = sessionSpy.prepare(s"SELECT * FROM $ks.test1").bind()
+      val statementCaptor = ArgumentCaptor.forClass(classOf[Statement[_]])
+
+      scanner.scan(stmt)
+      verify(sessionSpy).executeContinuously(statementCaptor.capture())
+      statementCaptor.getValue
+    } finally {
+      cqlSession.close()
+    }
+  }
+
+  it should "apply MB/s throughput limit" in skipIfNotDSE(conn) {
+    val executedStmt = executeContinuousPagingScan(ReadConf(throughputMiBPS = Some(32.0)))
+
+    executedStmt.getExecutionProfile.getBoolean(DseDriverOption.CONTINUOUS_PAGING_PAGE_SIZE_BYTES) should be(true)
```

Comment:
Nit: space after `should be` or not?

```
should be (true)
should be (1000)
```
or
```
should be(true)
should be(1000)
```

Reply:
I prefer the latter

```
should be(true)
should be(1000)
```
Grep shows that we are not consistent in this, `should be(` has more occurrences.